### PR TITLE
Making date bar sticky

### PR DESCRIPTION
**Pull request by votemike**
Did not bother with the JS polyfill as @BPScott said not too. See [his comment](https://jira.dev.bbc.co.uk/browse/PROGRAMMES-5590?focusedCommentId=3445365&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-3445365).

DO NOT MERGE while I test that {{ top: 0 }} doesn't break IE and EDGE.

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by votemike** [comment]
Implementing the sticky bar in this way leaves a 56px gap when using the period jump-links. Is that acceptable? Or should I re-implement the JS version of this?

**Comment by BPScott** [comment]
A realisation: Would it be worth using [`@supports`](https://developer.mozilla.org/en/docs/Web/CSS/@supports) to wrap the sticky nav logic, so that we don't end up applying a bunch css to browsers that don't support position:sticky?

**Comment by BjornTwachtmann** [comment]
I've tested it locally, seems to work pretty well. Lack of IE support might be an issue for some people, but we can re-visit it if they complain. 

I would say merge it.

File: src/Ds2013/Core/Modules/Schedules/_fixed-nav.scss [hunk 1]
```diff
@@ -5,15 +5,21 @@
...
+@supports (position: sticky) {
     $skiplink-pushdown: 56px;
 
     .schedule-nav-wrapper {
         top: 0;
         width: 100%;
         z-index: 1000;
+
+        position: -webkit-sticky;
+        position: -moz-sticky;
```
Comment (BPScott) [inline src/Ds2013/Core/Modules/Schedules/_fixed-nav.scss]: Taking a look at https://caniuse.com/#search=sticky it seems the only vendor prefixes that was ever used is -webkit, we can remove all the other prefixed versions

File: src/Ds2013/Molecule/DateList/_date-list.scss [hunk 2]
```diff
@@ -115,13 +115,6 @@
             }
         }
     }
-
-    // Don't pull upwards when fixed to the top of the page
-    .fixed-supported {
```
Comment (BPScott) [inline src/Ds2013/Molecule/DateList/_date-list.scss]: Would deleting break the positioning for tv schedules? It seems like this was reset for a reason

Comment (votemike) [inline src/Ds2013/Molecule/DateList/_date-list.scss]: It doesn't seem to have. What about it is different to radio?

**Pull request merged**